### DISCUSSION TITLE
OT_THREAD_VERSION_1_1 with radio support OT_RADIO_CAPS_TRANSMIT_SEC

### DISCUSSION THREAD

**Discussion by testter2** [openthread/openthread#8390] [Q&A] [answered]
Hi,

I'm asking when user chooses to use OT_THREAD_VERSION_1_1 with platform that support OT_RADIO_CAPS_TRANSMIT_SEC.

I'm just wondering if the radio platform should process security or it will be handled by Openthread ?

Thanks

**Comment by abtink** [comment, marked as answer]
If the radio platform indicates that it provides  `OT_RADIO_CAPS_TRANSMIT_SEC` capability, then OT core (`SubMac`) will not do tx security and delegates this to radio platform layer. This is independent of Thread version.

**Reply by testter2** [reply]
Thanks for quick reply.
